src/file.test.ts

```diff
@@ -1,15 +0,0 @@
-import { defaultSkynetPortalUrl, uriSkynetPrefix, SkynetClient } from "./index";
-
-const portalUrl = defaultSkynetPortalUrl;
-const client = new SkynetClient(portalUrl);
-
-describe("getEntryLink", () => {
-  it("Should get the entry link for a user ID and path", async () => {
```

Comment:
not sure why this got removed

Reply:
It was moved as-is to `integration.test.ts`. We should probably split that file up one day, though.